lib/mina/application.rb

```diff
@@ -14,9 +14,9 @@ def name
     end
 
     # :nocov:
-    def run
+    def run(*args)
```

Comment:
This is how run method [is defined](https://github.com/ruby/rake/blob/a842fb2c30cc3ca80803fba903006b1324a62e9a/lib/rake/application.rb#L79) in `Rake::Application`.